service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataService.java

```diff
@@ -56,7 +58,7 @@ public RoboticsData prepareRoboticsData(final CaseData caseData, final Long case
         return RoboticsData.builder()
             .caseNumber(caseData.getFamilyManCaseNumber())
             .applicationType(deriveApplicationType(caseData.getOrders()))
-            .feePaid(2055.00)
+            .feePaid(fromCCDMoneyGBP(caseData.getAmountToPay()).map(BigDecimal::doubleValue).orElse(2055.00))
```

Comment:
Can we keep it as BigDecimal or does it have to be converted to double? I think jackson will serialise it to the same number but would have to check. @Gogetter any suggestions?

Reply:
yes we can make this `BigDecimal` and use `@JsonFormat(shape = JsonFormat.Shape.NUMBER)` annotation on `feePaid` in `RoboticsData`. But the question is, do we want to make this change now and what would the value be?

Reply:
Well the value would be that BigDecimal is how you are supposed to store money in java however I do understand that with the operations we are doing (i.e. none) we won't run into any of the issues that come out of having it as a double

Reply:
Should we do any robotics testing if we make this change? I guess we are going to do it anyway!

Reply:
done